### PR TITLE
Add trustpay keys

### PR DESCRIPTION
**Pull request by Antti**
* Add trustpay keys

### CODE REVIEW

File: locales/bg.yml [hunk 1]
```diff
@@ -657,6 +661,14 @@ bg:
       last_name_sub_label: associated w/ credit card
       secure: Secure as Fort Knox and we don't store any of your personal information.
       title: Credit Card Payment
+    step3_cc_trustpay:
```
Comment (nielsenramon) [inline locales/bg.yml]: Should we also add the button copy in here (`Support FetLife Now`)? Or are we re-using the locale?

File: locales/de.yml [hunk 2]
```diff
@@ -563,6 +564,9 @@ de:
       cc_curo:
         extra_info: Wir akzeptieren Visa, V Pay, Mastercard und Maestro. Es ist leicht und sicher.
         title: Kreditkarte
+      cc_trustpay:
+        extra_info: We accept Visa, V Pay, Mastercard, and Maestro. It's easy and secure.
```
Comment (nielsenramon) [inline locales/de.yml]: Can we copy the german translation from above please?

File: locales/de.yml [hunk 3]
```diff
@@ -657,6 +661,14 @@ de:
       last_name_sub_label: mit der Kreditkarte verbunden
       secure: So sicher wie Fort Knox und wir speichern keine Deiner persönlichen Daten.
       title: Kreditkartenzahlung
+    step3_cc_trustpay:
+      cta: Support FetLife via Credit Card
+      first_name_label: First Name
```
Comment (nielsenramon) [inline locales/de.yml]: Do we need first and last name?

File: locales/bg.yml (hunk 1 above)
Comment (Antti) [reply locales/bg.yml]: 👍

File: locales/de.yml (hunk 3 above)
Comment (Antti) [reply locales/de.yml]: We don't. I've removed those

**Pull request merged**